### PR TITLE
Add nitro devnet deployed contracts

### PR DESCRIPTION
**Pull request by gzeoneth**

### CODE REVIEW

File: docs/Public_Nitro_Devnet.md [hunk 1]
```diff
@@ -55,3 +55,60 @@ For a deeper dive into deploying with truffle see [here](Contract_Deployment.md)
...
+| L1 Arb-Custom Gateway | [0x23D4e0D7Cb7AE7CF745E82262B17eb46535Ae819](https://goerli.etherscan.io/address/0x23D4e0D7Cb7AE7CF745E82262B17eb46535Ae819) |
+| L2 Arb-Custom Gateway | [0x7AC493f26EF26904E52fE46C8DaEE247b9A556B8](https://nitro-devnet-explorer.arbitrum.io/address/0x7AC493f26EF26904E52fE46C8DaEE247b9A556B8)  |
+| L1 Weth Gateway       | [0x64bfF696bE6a087A81936b9a2489624015381be4](https://goerli.etherscan.io/address/0x64bfF696bE6a087A81936b9a2489624015381be4) |
+| L2 Weth Gateway       | [0xf10c7CAA33A3360f60053Bc1081980f62567505F](https://nitro-devnet-explorer.arbitrum.io/address/0xf10c7CAA33A3360f60053Bc1081980f62567505F)  |
+| L1 Weth               | [0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6](https://goerli.etherscan.io/address/0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6) |
+| L2 Weth               | [0x96CfA560e7332DebA750e330fb6f59E2269f40Dd](https://nitro-devnet-explorer.arbitrum.io/address/0x96CfA560e7332DebA750e330fb6f59E2269f40Dd)  |
+| OneStepProver0      | [0x91ddccc4cccbb7609e045e10d311712789f2010f](https://goerli.etherscan.io/address/0x91ddccc4cccbb7609e045e10d311712789f2010f) |
```
Comment (yahgwai) [inline docs/Public_Nitro_Devnet.md:89]: These onestep prover contracts are run as part of the fraud proofs. Maybe they should be in the protocol section, or the misc section?

Comment (gzeoneth) [reply docs/Public_Nitro_Devnet.md:89]: Fine with either, we do list them in the useful addresses page. On the other hand, in Classic we have these as proxies, do we have the same with Nitro?

File: docs/Public_Nitro_Devnet.md [hunk 2]
```diff
@@ -55,3 +55,60 @@ For a deeper dive into deploying with truffle see [here](Contract_Deployment.md)
...
+| OneStepProverMemory | [0x4A78961010c3A6c587077Efc0CD6BaA22c974E0a](https://goerli.etherscan.io/address/0x4A78961010c3A6c587077Efc0CD6BaA22c974E0a) |
+| OneStepProverMath   | [0x8b6dBCA5A16d77333819487143Ad88653E5D2574](https://goerli.etherscan.io/address/0x8b6dBCA5A16d77333819487143Ad88653E5D2574) |
+| OneStepProverHostIo | [0xB4dba7DB08fBcF1809ec4A2139d168Fa3f466868](https://goerli.etherscan.io/address/0xB4dba7DB08fBcF1809ec4A2139d168Fa3f466868) |
+| OneStepProofEntry   | [0x1c6bA2a2c079Df3608961135E1cdd65d908AE23e](https://goerli.etherscan.io/address/0x1c6bA2a2c079Df3608961135E1cdd65d908AE23e) |
+
+### Token Bridge
+
+**IMPORTANT**: _Do **not** simply transfer tokens or Ether to any of the addresses below; it will result in loss of funds._
+
+_Users should only interact with the token bridge via dapp interfaces like https://nitro-devnet-bridge.arbitrum.io.
```
Comment (fredlacs) [inline docs/Public_Nitro_Devnet.md:80]:
```suggestion
Users should only interact with the token bridge via dapp interfaces like https://nitro-devnet-bridge.arbitrum.io.
```

**Review by fredlacs — APPROVED** [review]
LGTM

**Pull request merged**